roles/cdi/defaults/main.yml

```diff
@@ -1,8 +1,9 @@
-cdi_namespace: "{{ cdi_image_namespace | default('kube-system') }}"
+---
+cdi_namespace: "{{ cdi_image_namespace | default('cdi') }}"
 cdi_storageclass: "{{ cdi_kubevirt_storageclass | default('kubevirt') }}"
 apb_action: "provision"
 docker_prefix: "{{ cdi_repo_tag | default('kubevirt') }}"
-cdi_release_tag: "v1.4.0"
+cdi_release_tag: "latest"
```

Comment:
v1.5.1 was just published, so let's move over to that.

Reply:
done